### ISSUE TITLE
Recharge Planters not working.

### ISSUE THREAD

**Issue by Poofey** [goatfungus/NMSSaveEditor#141]
I may be doing it wrong but 
1. i've hit the button and saved changes
2. made sure i've selected the base i want in the bases/storage section

The trays (either the single or quad ones) remain empty. Am i missing something or is this legit not working?

Also, thanks for this tool, it's legit saved my save file a couple of times now.

**Comment by goatfungus** [comment]
The recharge function in the game doesn't function properly. Sometimes planters don't ever need to be charged, and sometimes they do. For this reason I haven't managed to get the recharge to work properly in the editor either.

**Comment by Poofey** [comment]
Yeah, I've noticed that the planters are sometimes full and sometines empty.

**Comment by ukmerlin** [comment]
Just adding that it happens to me too and that sometimes if I recharge a planter in game the plant will instantaneously be ready for harvesting.
